src/test/java/com/easyrestaurant/tests/CreateAdministratorProfileTest.java

```diff
@@ -0,0 +1,74 @@
+package com.easyrestaurant.tests;
+
+import java.sql.SQLException;
+
+
+import com.easyrestaurant.core.TestUtils;
+import com.easyrestaurant.utils.CheckRestaurantExist;
+import com.easyrestaurant.utils.LoadProperties;
+import io.qameta.allure.Description;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class CreateAdministratorProfileTest extends TestUtils {
+	
+//	WebDriver driver;
+	static LoadProperties properties = new LoadProperties();
+//	static CheckRestaurantExist restaurant = new CheckRestaurantExist();
+
+	@Parameters({"username", "password"})
+	@Description("Create Administrator Profile From owner's menu")
+	@Test(description = "Create Administrator Profile From owner's menu",
+			priority = 7)
+	public void createWaiterProfile() throws SQLException {
```

Comment:
The name of the method is "createWaiterProfile" and the name of the class is "CreateAdministratorProfile", it's confusing. Please change the method name, because the test creates administrator profile.

Reply:
It is fixed now.